exercises/sam_jenkins/ex02_oo_basics/src/Square.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Shapes.h"
+#include "Square.h"
+	Square::Square() :Shapes("Square", 4), side(1)
+	{
+
+	}
+	Square::Square(double side) :Shapes("Square", 4), side(side)
```

Comment:
A better name would than `side` would be `length` or `edgeLength` so it's not confusing with `sides`